py4DSTEM/braggvectors/diskdetection_aiml.py

```diff
@@ -254,9 +250,7 @@ def find_Bragg_disks_aiml_selected(
         correlation intensities at each scan position (Rx,Ry).
     """
 
-    try:
-        import crystal4D
-    except:
+    if find_spec("crystal4D") is None:
```

Comment:
Crystal4D is required to load the model, so on line 260 it will download the model, do some other stuff, and eventually try and load the model and fail. I think the error is a little obtuse. So its quicker to check if its install and raise a more descriptive error.

Reply:
If a `try...except` works to catch the weird error message and output a reasonable one @alex-rakowski I think this is nicer for consistency